Package.resolved

```diff
@@ -77,8 +77,8 @@
         "package": "Nimble",
         "repositoryURL": "https://github.com/Quick/Nimble",
         "state": {
-          "branch": "master",
-          "revision": "eea5843b34beb559dd51cf004953f75028e47add",
+          "branch": "main",
```

Comment:
Let's lock into a version `.upToNextMajor`

Reply:
I have tried it but it gives me error because it does not find the library: `NIOConcurrencyHelpers`. I have left it in the main branch in the meantime.